unity-renderer/Assets/Rendering/LoadingAvatar/Editor/BaseAvatarTest.cs

```diff
@@ -0,0 +1,18 @@
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+
+namespace Rendering.LoadingAvatar.Editor
+{
+    public class BaseAvatarTest
```

Comment:
```suggestion
    [TestFixture]
    [Category("EditModeCI")]
    public class BaseAvatarTest
```

Remember to add the EditModeCI category, otherwise is gonna be ignored by the CI